runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -6061,7 +6017,7 @@ mod altair {
 					orml_tokens::Pallet::<T>::free_balance(usdc_asset_id, &Keyring::Bob.into());
 
 				// Sanity check to ensure the calculated is what is expected
-				assert_eq!(bob_balance, 11992961);
+				assert_eq!(bob_balance, 11993571);
```

Comment:
?

Reply:
See the thread on this PR: https://github.com/centrifuge/centrifuge-chain/pull/1776#issuecomment-2030282314

Reply:
Some fee computations changed.